src/java.base/share/classes/java/security/cert/LDAPCertStoreParameters.java

```diff
@@ -31,9 +31,9 @@
  * This class is used to provide necessary configuration parameters (server
  * name and port number) to implementations of the LDAP {@code CertStore}
  * algorithm. However, if you are retrieving certificates or CRLs from
- * an ldap URI as specified by RFC 5280, use the
+ * a ldap URI as specified by RFC 5280, use the
```

Comment:
Isn't `ldap` pronounced `L-Dap`?

Reply:
You are right. IntelliJ was wrong and I didn't catch it.